.travis.yml

```diff
@@ -15,22 +15,21 @@ addons:
   apt:
     packages:
     - wget
+env:
+  global:
+    - CACOPHONY_VER=v4.10.0  # 2019-05-08 https://github.com/TheCacophonyProject/cacophony-api/releases/tag/v4.10.0   0c7e3bf14a9464e36dc8146a72d7e13a5441f34f
```

Comment:
How about calling this `API_VER` or `CACOPHONY_API_VER` to make it clearer what it's about?

Reply:
CACOPHONY_API_VER makes sense, will change.
I have some questions worth discussion on how to version the client and how to sync versions and what tests can cover api tests simultaneous to python and go clients. Basically trying to avoid multiple instances of code, This is possibly a Trello card.

Reply:
I think this is ready to merge once you've made the CACOPHONY_API_VER change.

Let's discuss versioning via a call.

Reply:
Done, and raised #4 to capture further thoughts